app/views/content_items/guide.html.erb

```diff
@@ -2,11 +2,35 @@
 <div class="grid-row">
   <div class="column-two-thirds">
     <%= render 'govuk_component/title', { title: @content_item.title } %>
-    <%= render 'guide_body' %>
+    <% if @content_item.multi_page_guide? %>
+      <aside class="part-navigation-container" role="complementary">
+        <%= render 'shared/parts_navigation', content_item: @content_item, navigation_label: 'Pages in this guide' %>
+      </aside>
+    <% end %>
+  </div>
+  <div class="column-two-thirds">
+    <% if @content_item.has_parts? %>
+      <% if @content_item.multi_page_guide? %>
+        <h1 class="part-title">
+          <%= @content_item.current_part_title_with_index %>
+        </h1>
+      <% end %>
+
+      <%= render 'govuk_component/govspeak',
+          content: @content_item.current_part_body,
+          direction: page_text_direction,
+          disable_youtube_expansions: true,
+          rich_govspeak: true %>
+      <%= render 'govuk_component/previous_and_next_navigation', @content_item.previous_and_next_navigation %>
+
+      <% if @content_item.multi_page_guide? %>
+        <%= render 'components/print-link', href: @content_item.print_link, link_text: t("multi_page.print_entire_guide") %>
+      <% end %>
+    <% end %>
   </div>
   <% if current_tasklist_ab_test.show_tasklist_sidebar? %>
```

Comment:
`shared/sidebar_navigation` already contains this check, so I think you should be able to delete this block and just import `shared/sidebar_navigation`?